modules/eco-poison-pill-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * /machine_management/eco-poison-pill-operator.adoc
+
+[id="troubleshooting-poison-pill-operator_{context}"]
+= Troubleshooting the Poison Pill Operator
+
+
+[id="general-troubleshooting-poison-pill-operator_{context}"]
+== General Troubleshooting 
+
+Issue::
+You want to troubleshoot issues with the Poison Pill Operator.
+
+Resolution::
+Enable the debug flag in the `poison-pill` object. The `debug` flag increases
```

Comment:
The example CR in "Understanding the Poison Pill Operator configuration" does not list a "debug" flag.

Reply:
Updated the resolution section to check pod logs.